dotcom-rendering/src/components/ArticleMeta.apps.tsx

```diff
@@ -0,0 +1,351 @@
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDesign, ArticleSpecial } from '@guardian/libs';
+import {
+	between,
+	border,
+	from,
+	space,
+	until,
+} from '@guardian/source-foundations';
+import { StraightLines } from '@guardian/source-react-components-development-kitchen';
+import { interactiveLegacyClasses } from '../layouts/lib/interactiveLegacyStyling';
+import { getSoleContributor } from '../lib/byline';
+import { palette as themePalette } from '../palette';
+import type { Branding as BrandingType } from '../types/branding';
+import type { TagType } from '../types/tag';
+import {
+	metaContainer as metaContainerMargins,
+	shouldShowAvatar,
+	shouldShowContributor,
+} from './ArticleMeta';
+import { Avatar } from './Avatar';
+import { Branding } from './Branding.importable';
+import { CommentCount } from './CommentCount.importable';
+import { Contributor } from './Contributor';
+import { Dateline } from './Dateline';
+import { FollowWrapper } from './FollowWrapper.importable';
+import { Island } from './Island';
+import { SendAMessage } from './SendAMessage.importable';
+
+type Props = {
+	format: ArticleFormat;
+	pageId: string;
+	webTitle: string;
+	byline?: string;
+	tags: TagType[];
+	primaryDateline: string;
+	secondaryDateline: string;
+	branding?: BrandingType;
+	discussionApiUrl: string;
+	shortUrlId: string;
+	isCommentable: boolean;
+	ajaxUrl: string;
+	messageUs?: MessageUs;
+};
+
+const metaGridContainer = (showAvatar: boolean) => css`
+	display: grid;
+	grid-template-rows: auto;
+
+	${showAvatar
+		? css`
+				grid-template-columns: auto 1fr auto;
+				grid-template-areas:
+					'avatar byline comment-count'
+					'avatar byline comment-count'
+					'dateline dateline dateline';
+		  `
+		: css`
+				grid-template-columns: 1fr auto;
+				grid-template-areas:
+					'byline comment-count'
+					'byline comment-count'
+					'dateline dateline';
+		  `}
+`;
+
+const MetaGridAvatar = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			grid-area: avatar;
+			width: 140px;
+			height: 140px;
+			margin-top: 6px;
+			margin-right: 10px;
+			margin-bottom: 12px;
+			margin-left: 0px;
+
+			${until.leftCol} {
+				grid-area: avatar;
+				width: 60px;
+				height: 60px;
+				margin-top: 3px;
+				margin-right: 10px;
+				margin-bottom: 12px;
+				margin-left: 0px;
+			}
+		`}
+	>
+		{children}
+	</div>
+);
+
+const MetaGridByline = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			grid-area: byline;
+			width: auto;
+		`}
+	>
+		{children}
+	</div>
+);
+
+const MetaGridCommentCount = ({
+	children,
+	isPictureContent,
+}: {
+	children: React.ReactNode;
+	isPictureContent: boolean;
+}) => (
+	<div
+		data-print-layout="hide"
+		css={css`
+			grid-area: comment-count;
+			border-left: 1px solid ${themePalette('--article-border')};
+
+			${until.desktop} {
+				margin-top: ${isPictureContent ? '-6px' : '-2px'};
+				padding-top: ${isPictureContent ? '6px' : '2px'};
```

Comment:
What is the purpose of this?

Reply:
It's so the left border hits the bottom horizantal line on the CommentCount and the PictureLayout was different. 

<img width="348" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/32312712/7322cc9f-3e18-4619-b031-2a2004cf48f1">


<img width="402" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/32312712/ee54645e-ccbb-4c59-953e-65910db46440">